### ISSUE TITLE
Problem with Sollet.io account ?

### ISSUE THREAD

**Issue by KebapMaster** [project-serum/spl-token-wallet#230]
Hi, in order to be involved with SPL token whitelist, I created a sollet account and created a new account (not main account) I transferred my SOL to this account but while I was doing KYC, I froze and I shut the page down and opened again but there is not that account and I cant see my money.. Anyone who can help me? thanks

**Comment by GentlemansKiller** [comment]
@KebapMaster Did you backup your seed phrase? That's your key to your solana funds on the blockchain, you can restore from this seed

**Comment by KebapMaster** [comment]
Yes, I have my seed but there is not the address which I sent on the accounts list on the right top of the website. When I check the lists of address on the restore alternatives, there is not either...

**Comment by GentlemansKiller** [comment]
You might try pasting the different addresses into the Solana explorer and see if sollet is just not picking up the funds. When you paste in an address, you can also look under the "Tokens" tab
https://explorer.solana.com/

Thats the only thing i can think of... If you create a new account/address from a seed it should just pick the next one in the list

Curiously, how much solana were you handling? if it's ok to ask..

**Comment by GentlemansKiller** [comment]
Also, use the solana explorer to paste in the address where the SOL came from, and see the transactions on that address

**Comment by moondoll31** [comment]
> Also, use the solana explorer to paste in the address where the SOL came from, and see the transactions on that address

I have 0.24 sol in my account solana explorer shows my account and the transaction i did but on sollet.io when I add seed pharse to restore account I get a new wallet address with 0 funds in it.i created 13 new wallets using al 3 derivatives paths but failed

**Comment by samhau2020** [comment]
I just deleted my history in the chrome and have to restore the sollet wallet, but cannot remember the phrase words, only got the password, how can I access my wallet?
